### PR TITLE
Added WWN to go brick volumeInfo 

### PR DESCRIPTION
**Pull request by francis-nijay**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,8 @@ module github.com/dell/csi-powerstore
 
 go 1.17
 
+replace github.com/dell/gobrick => ../gobrick
```
Comment (spriya-m) [inline go.mod:5]: please remove reference to local directory

Comment (harishp8889) [reply go.mod:5]: @harshitap26 Please use the gonvme `main` branch commit id with `go get@<commit>`

**Review by harishp8889 — CHANGES_REQUESTED** [review]
Please make sure to update the below things in the PR:
1. Attach the Test results
2. Make sure to tick all the checks in the template
3. Mention the tests you have done in the template (Test A, Test B)

**Comment by francis-nijay** [comment]
Closing the PR since we have alternate approach to achieve the requirement.

**Pull request closed without merging**